fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/FluentIcon.kt

```diff
@@ -14,7 +20,8 @@ data class FluentIcon(
     val dark: ImageVector = light,
     val contentDescription: String? = null,
     val onClick: (() -> Unit)? = null,
-    val tint: Color? = null
+    val tint: Color? = null,
+    val flipOnRtl: Boolean = true
```

Comment:
put onClick at last so that it could use kotlin syntax to create function. Also flipOnRtl should be false by default.